doc/source/data/batch_inference.rst

```diff
@@ -1,823 +1,455 @@
-
-1. **Faster and Cheaper for modern Deep Learning Applications**:
-    Ray is built for
-    complex workloads and supports loading and preprocessing data with CPUs and model inference on GPUs.
-2. **Cloud, framework, and data format agnostic**:
-    Ray Data works on any cloud provider or
-    any ML framework (like PyTorch and Tensorflow) and does not require a particular file format.
-3. **Out of the box scaling**:
-    The same code that works on one machine also runs on a
-    large cluster without any changes.
-4. **Python first**:
-    You can express your inference job directly in Python instead of
-    YAML files or other formats.
-
-.. _batch_inference_quickstart:
-
-Quick Start
------------
-
-If you're impatient and want to see a copy-paste example right away,
-here are a few simple examples.
-Just pick one of the frameworks you like and run the code in your terminal.
-If you want a more detailed rundown of the same examples, skip ahead to the
-:ref:`following batch inference walk-through with Ray<batch_inference_walk_through>`.
-
-
-.. tabs::
-
-    .. group-tab:: HuggingFace
-
-        .. literalinclude:: ./doc_code/hf_quick_start.py
-            :language: python
-            :start-after: __hf_super_quick_start__
-            :end-before: __hf_super_quick_end__
-
-    .. group-tab:: PyTorch
 
-        .. literalinclude:: ./doc_code/pytorch_quick_start.py
-            :language: python
-            :start-after: __pt_super_quick_start__
-            :end-before: __pt_super_quick_end__
+Why choose Ray Data for offline inference?
+------------------------------------------
 
-    .. group-tab:: TensorFlow
+.. dropdown:: Faster and cheaper for modern deep learning applications
 
-        .. literalinclude:: ./doc_code/tf_quick_start.py
-            :language: python
-            :start-after: __tf_super_quick_start__
-            :end-before: __tf_super_quick_end__
+    Ray Data is for deep learning applications that involve both CPU preprocessing and GPU inference. Ray Data streams the working data from CPU preprocessing tasks to GPU inferencing tasks, allowing you to utilize both sets of resources concurrently.
 
+    By using Ray Data, your GPUs are no longer idle during CPU computation, reducing overall cost of the batch inference job.
 
-.. _batch_inference_walk_through:
+.. dropdown:: Cloud, framework, and data format agnostic
 
-Walk-through: Batch Inference with Ray
---------------------------------------
+    Ray Data has no restrictions on cloud provider, ML framework, or data format.
+    
+    Through the :ref:`Ray cluster launcher <cluster-index>`, you can start a Ray cluster on any of the popular cloud providers. Since Ray Data works with any Python function or class, you can use any ML framework of your choice, including PyTorch, HuggingFace, or Tensorflow. Ray Data also does not require a particular file format, and supports a :ref:`wide variety of formats <loading_data>` including CSV, Parquet, raw images, etc.
```

Comment:
```suggestion
    Through the :ref:`Ray cluster launcher <cluster-index>`, you can start a Ray cluster on any of the popular cloud providers. Since Ray Data works with any Python function or class, you can use any ML framework of your choice, including PyTorch, HuggingFace, or Tensorflow. Ray Data also does not require a particular file format, and supports a :ref:`wide variety of formats <loading_data>` including CSV, Parquet, and raw images.
```

> Avoid using etc., and so forth, and and so on wherever possible. If you really need to use one, use etc. Always include the period, even if a comma follows immediately after.

https://developers.google.com/style/word-list